articles/azure-stack/azure-stack-powershell-install.md

```diff
@@ -83,7 +83,7 @@ Before installing the required version, make sure that you uninstall any previou
 
 ## 4. Connected: Install PowerShell for Azure Stack with Internet connectivity
 
-Azure Stack requires the **2018-03-01-hybrid** API version profile for Azure Stack version 1808 or later. The profile is available by installing the **AzureRM.BootStrapper** module. In addition, to the AzureRM modules, you should also install the Azure Stack-specific PowerShell modules. The API version profile and Azure Stack PowerShell modules you require will depend on the version of Azure Stack your are running.
+Azure Stack requires the **2018-03-01-hybrid** API version profile for Azure Stack version 1808 to 1811. The profile is available by installing the **AzureRM.BootStrapper** module. For Azure Stack version 1901 or later, please use AzureRM version 2.4.0. **AzureRM.Bootstrapper** is not needed to install the modules for Azure stack version 1901 or later. In addition, to the AzureRM modules, you should also install the Azure Stack-specific PowerShell modules. The API version profile and Azure Stack PowerShell modules you require will depend on the version of Azure Stack your are running.
```

Comment:
im not sure how to reword this.. this somehow gives a message that we are moving away from profiles